package.json

```diff
@@ -4,11 +4,13 @@
   "description": "Arbitration Outsourcing for State Channels",
   "main": "index.js",
   "scripts": {
-    "test": "npx tsc && mocha build/__pisa_tests__ --exit",
+    "test": "npm run build && mocha $(find ./build/__pisa_tests__/*.js ! -path '*quickTest*') --exit",
+    "test-docker": "mocha ./build/__pisa_tests__/quickTest.js --exit",
```

Comment:
please dont use ./ when referring to a folder/file which is a sibling

Reply:
kk will do